driver/config/config_test.go

```diff
@@ -144,7 +144,7 @@ func TestViperProvider(t *testing.T) {
 				config  string
 				enabled bool
 			}{
-				{id: "password", enabled: true, config: `{"haveibeenpwned_host":"api.pwnedpasswords.com","ignore_network_errors":true,"max_breaches":0}`},
+				{id: "password", enabled: true, config: `{"haveibeenpwned_host":"api.pwnedpasswords.com","haveibeenpwned_enabled":true,"ignore_network_errors":true,"max_breaches":0}`},
```

Comment:
Add a test case with false here :)

Reply:
My understanding is that "group=methods" test case only validates config defaults; it does not allow for specifying inputs to new configs. I've added new unit test coverage for all of the haveibeenpwned settings.